src/config.ts

```diff
@@ -41,6 +41,7 @@ export const configConstants = {
     prefix: "sls",
     localPort: 7071,
   },
+  bindingsJsonUrl: "https://raw.githubusercontent.com/Azure/azure-functions-templates/master/Functions.Templates/Bindings/bindings.json"
```

Comment:
Can you update our default file with the latest from the remote version?

Reply:
Sorry, I should've specified in the description, that's included. It's a huge file, so Github doesn't show the diff by default

Reply:
Ah, i thought it would have shown...